static/js/components/ConfirmationModal.tsx

```diff
@@ -40,6 +40,12 @@ const ConfirmationModal: FC<Props> = ({
       <p style={{ textAlign: "start", whiteSpace: "pre-line" }}>
         {confirmationMessage}
       </p>
+      <p>
```

Comment:
Maybe a bit lighter w/o the bold text? I.e. 

```

      <p className="u-text--muted">
        You can skip these confirmation modals by holding <code>SHIFT</code>{" "}
        when clicking on the action.
      </p>
```

Reply:
Done @ [7144eb8](https://github.com/canonical/lxd-ui/commit/7144eb842a49e2eb2afca0173f072e895669bab0)